autopilot/sagemaker_autopilot_neo4j_portfolio_churn.ipynb

```diff
@@ -0,0 +1,916 @@
+{
```

Comment:
<p>Change the order to run the drop command before the create command? Otherwise if a user runs all the cells, the table will be created and then dropped.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3505/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
<p>That'll throw a pretty gnarly error.  I'm going to comment it out instead if that's ok.  I'll leave this one open until you confirm that is ok.</p>

Reply:
<p>Commenting out works for me.</p>